funlet-simple-message/.env

```diff
@@ -1,3 +1,24 @@
+# description: The first URL to <Play>, or string to <Say>.
+# format: text
+# required: true
+FUNLET_MESSAGE1=Hello, world!
+
+# description: The second URL to <Play>, or string to <Say>.
+# format: text
+FU1LET_MESSAGE2=
```

Comment:
why are these FU1Let?

Reply:
I'm honestly not sure how that happened; fixed.